modules/telco-troubleshooting-security-authentication.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-security.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="telco-troubleshooting-security-authentication_{context}"]
+= Authentication
+
+Determine which identity provider or providers are in your cluster.
+For more information about supported identity providers, see "Supported identity providers" in _Authentication and authorization_.
+
+After you know which provider or providers are configured, you can inspect the `openshift-authentication` namespace to determine where potential issues might be.
```

Comment:
```suggestion
After you know which providers are configured, you can inspect the `openshift-authentication` namespace to determine if there are potential issues.
```
Or consider "determine if potential issues exist" or something similar. I think "might be" is discouraged by IBM Style although I can't find a specific reference.